tests/test_salted.py

```diff
@@ -0,0 +1,64 @@
+from tempfile import TemporaryDirectory
+from unittest import TestCase
+import os
+import luigi
+
+from pset_5 import salt
+
+
```

Comment:
Great that you managed to implement the salted version. I ran out of time. I will have a more detailed look into this after the course is finished.